easybuild/easyblocks/c/clang.py

```diff
@@ -113,6 +116,31 @@ def __init__(self, *args, **kwargs):
         self.llvm_obj_dir_stage3 = None
         self.make_parallel_opts = ""
 
+        if not self.cfg['llvm_projects']:
+            self.cfg['llvm_projects'] = []
```

Comment:
This could be the place to check
```
            self.cfg['llvm_projects'] = []
        else:
            for project in self.cfg['llvm_projects']:
                if project not in KNOWN_LLVM_PROJECTS:
                    warn_msg = "LLVM project %s included but not recognised, this project will NOT be sanity checked!" % project
                    self.log.warning(warn_msg)
                    print_warning(warn_msg)
```